tests/jenkins/run_test.sh

```diff
@@ -6,17 +8,38 @@ echo "USE_CUDNN=1" >> config.mk
 echo "USE_PROFILER=1" >> config.mk
 echo "DEV=1" >> config.mk
 echo "EXTRA_OPERATORS=example/ssd/operator" >> config.mk
+user=`id -u -n`
 make -j$(nproc) || exit -1
 
+echo "BUILD python2 mxnet"
+cd python
+if [ $user == 'root' ]
+then
+    python setup.py install || exit 1
```

Comment:
@lxn2 This messed up the build server and broke the build.
Please don't modify run_test.sh unless you are certain about the consequences.